Libplanet/Store/IStore.cs

```diff
@@ -231,6 +231,42 @@ void PutBlock<T>(Block<T> block)
         /// <seealso cref="PutTxExecution(Libplanet.Tx.TxSuccess)"/>
         TxExecution GetTxExecution(BlockHash blockHash, TxId txid);
 
+        /// <summary>
+        /// Records a index for given pair <paramref name="txId"/> and <paramref name="blockHash"/>.
+        /// If there exist a record for <paramref name="txId"/> already,
+        /// it overwrites the record silently.
+        /// </summary>
+        /// <param name="chainId">The chain ID of the index.</param>
+        /// <param name="txId">The <see cref="TxId"/> of the <see cref="Transaction{T}"/>.</param>
+        /// <param name="blockHash">The <see cref="BlockHash"/> of the <see cref="Block{T}"/>.
+        /// </param>
+        public void PutTxIdBlockHashIndex(Guid chainId, TxId txId, BlockHash blockHash);
+
+        /// <summary>
+        /// Determines whether the <see cref="IStore"/> contains index to <see cref="BlockHash"/>
+        /// with given <paramref name="txId"/>.
+        /// </summary>
+        /// <param name="chainId">The chain ID of the index.</param>
+        /// <param name="txId">The <see cref="TxId"/> of the <see cref="Transaction{T}"/>.</param>
+        /// <returns><c>true</c> if the index with given <paramref name="txId"/> is exists.
+        /// Otherwise, <c>false</c>.</returns>
+        public bool HasTxIdBlockHashIndex(Guid chainId, TxId txId);
+
+        /// <summary>
+        /// Retrieves the <see cref="BlockHash"/> indexed by the <paramref name="txId"/>.
+        /// </summary>
+        /// <param name="chainId">The chain ID of the index.</param>
+        /// <param name="txId">The <see cref="TxId"/> of the <see cref="Transaction{T}"/>.</param>
+        /// <returns><see cref="BlockHash"/> if the index exists. Otherwise <c>null</c>.</returns>
+        public BlockHash? GetTxIdBlockHashIndex(Guid chainId, TxId txId);
+
+        /// <summary>
+        /// Delete the index for the <paramref name="txId"/>.
```

Comment:
[We tend to use third person singular verbs for XML comments.][1]

```suggestion
        /// Deletes the index for the <paramref name="txId"/>.
```

[1]: https://weblogs.asp.net/dixin/csharp-coding-guidelines-6-documentation